test/e2e/framework/upgmode.go

```diff
@@ -240,6 +240,10 @@ func tdfVPPConfigIPv4() vpp.VPPConfig {
 						Dst: MustParseIPNet("10.1.0.0/16"),
 						Gw:  MustParseIP("10.0.1.2"),
 					},
+					{
```

Comment:
It would be nice to add a comment before this line e.g.
```
// used by NAT tests
```

Reply:
Comment has been added, thanks